File: src/main/java/me/yoursole/ctf/Events/Join.java
package me.yoursole.ctf.Events;

import me.yoursole.ctf.DataFiles.GameData;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class Join implements Listener {

    @EventHandler
    public void onPlayerJoin(PlayerJoinEvent e){
        GameData.scores.put(e.getPlayer().getUniqueId(),0);
        e.getPlayer().discoverRecipes(GameData.recipeKeys.values());
    }
}

File: src/main/kotlin/me/yoursole/ctf/events/Join.kt
package me.yoursole.ctf.events

import me.yoursole.ctf.datafiles.GameData
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

object Join : Listener {
    @EventHandler
    fun onPlayerJoin(e: PlayerJoinEvent) {
        GameData.scores[e.player.uniqueId] = 0
        e.player.discoverRecipes(GameData.recipeKeys.values)
    }
}